script.js

```diff
@@ -27,68 +146,97 @@ document
   .getElementById("addRecipeBtn")
   .addEventListener("click", addRecipeClick);
 
-//*******************  show recipes in the page **********************/
+document.getElementById("searchBtn").addEventListener("click", searchClicked);
+
+document.getElementById("sortAscBtn").addEventListener("click", sortAscClicked);
+
+document.getElementById("sortDesBtn").addEventListener("click", sortDesClicked);
+
+//********************* show recipes in grid **********************/
+
+function showRecipesInGrid() {
+  for (let i = 0; i < recipes.length; i++) {
```

Comment:
This is fine and acceptable. However, it's nice to get used to the cleaner approach where you can easily avoid mistakes related to the comparison operator `<`, or forgetting to increment `i++`
```suggestion
  recipes.forEach((recipe) => {
```
then you will replace `recipes[i]` by `recipe`